src/app/core/navbar/navbar.component.html

```diff
@@ -1,14 +1,39 @@
 <div class="navContainer">
-    <div class="logoSection" [routerLink]="['/']" >
-        <div class="hamburger hamburger--spin hideDesktop"
-        (click)="hamburgerIsActive=!hamburgerIsActive" 
-        [ngClass]="hamburgerIsActive ? 'is-active hamburgerTurn' : ''">
+    <div class="logoSection" [routerLink]="['/']">
+        <div class="hamburger hamburger--spin hideDesktop" (click)="hamburgerIsActive=!hamburgerIsActive"
+            [ngClass]="hamburgerIsActive ? 'is-active hamburgerTurn' : ''">
             <div class="hamburger-box">
-              <div class="hamburger-inner"></div>
+                <div class="hamburger-inner"></div>
             </div>
-          </div>
+        </div>
 
-        <svg class="verticalAlign hideMobile" data-name="ripeLogo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 245.16 365.82"><defs><style>.cls-1{fill:#fff;}.cls-2{fill:#2366af;}</style></defs><title>HackathonLogo</title><path class="cls-1" d="M375.15,306.18c27.48-3.62,38.83,2.56,49.84,26.31a191,191,0,0,1,9.62,24.85A182.12,182.12,0,0,0,462.41,413c23.36,31.75,31.28,67.75,19.11,105.13-26.26,80.69-113.62,90-165.38,57.88-27.4-17-46.45-38.58-49.51-72.13-2.46-26.94,6.07-50.5,19.68-72.94,6.56-10.82,13.75-21.43,18.84-32.94a247.21,247.21,0,0,0,14.31-41.49c5.39-22.33,15.42-39.44,39.83-44.07,7.35-1.4,6.92-6.62,6.77-12.39-.35-13.24-.11-26.5-.11-40.37h7C373.67,275.44,374.41,290.72,375.15,306.18Z" transform="translate(-242.02 -227.39)"/><path class="cls-1" d="M357,295.59c-25.45,16.17-46.14-.69-67.86-8.58-5.68-2.07-9.77-8.21-15.09-11.7-5.49-3.6-12.43-5.31-17.23-9.55-5.76-5.09-9.92-12-14.79-18.11,12.63-6.28,23.22-13.43,34.91-17,31-9.43,49.29,2.28,67.39,25.31C353.59,267.77,358.35,281.61,357,295.59Z" transform="translate(-242.02 -227.39)"/><path class="cls-2" d="M375.4,329.1c23-3.05,32.47,2.16,41.69,22.18a162.57,162.57,0,0,1,8,21,154,154,0,0,0,23.25,46.92c19.53,26.77,26.15,57.13,16,88.65-22,68-95,75.91-138.3,48.81-22.92-14.34-38.85-32.53-41.41-60.82-2.06-22.72,5.07-42.59,16.45-61.51,5.49-9.13,11.51-18.08,15.76-27.78a210.14,210.14,0,0,0,12-35c4.51-18.83,12.9-36.25,33.31-40.16" transform="translate(-242.02 -227.39)"/><path class="cls-2" d="M343.1,286.29s-11-25-37-37-47,0-47,0,28.06-.88,41,5C311.1,259.29,313.1,257.29,343.1,286.29Z" transform="translate(-242.02 -227.39)"/></svg>
+        <svg class="verticalAlign hideMobile" data-name="ripeLogo" xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 245.16 365.82">
+            <defs>
+                <style>
+                    .cls-1 {
+                        fill: #427AA1;
+                    }
+
+                    .cls-2 {
+                        fill: #6EA0C4;
+                    }
+                </style>
+            </defs>
```

Comment:
Is there a reason why you put the css here?

Reply:
This is your SVG from the hackathon. The diff is from me saving the file and my autoFormatter. That is a good point though and I'll go put it in the style sheet.